app/services/webhooks/add_on_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Webhooks
+  class AddOnService < Webhooks::BaseService
+    private
+
+    def current_organization
+      @current_organization ||= object.organization
+    end
+
+    def object_serializer
+      ::V1::InvoiceSerializer.new(
+        object,
+        root_name: 'invoice',
+        includes: %i[customer subscription],
```

Comment:
Following my comments about fees, you should also include fees in the webhook

It will require an update of the fee model on:
- https://github.com/getlago/lago-api/blob/main/app/models/fee.rb#L37
- https://github.com/getlago/lago-api/blob/main/app/models/fee.rb#L43
- https://github.com/getlago/lago-api/blob/main/app/models/fee.rb#L49

Reply:
Tnx for the comments, updated